typing/base.go

```diff
@@ -1,10 +1,13 @@
 package typing
 
-type Health struct {
-	Version     string `json:"version"`
+type Home struct {
 	Status      bool   `json:"status"`
+	Version     string `json:"version"`
 	Description string `json:"description"`
 }
+type Health struct {
```

Comment:
Please, let there be a line spacing between the structs.

Also, why do we have just `Status` in the health struct? Isn't there any other health information to be returned?

Eg... `Name`, `Version`